libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1025,3 +1025,61 @@ fn temp_indent() {
 )"#,
     );
 }
+#[test]
+fn else_indentation() {
+    assert_unchanged(
+        "tables
+    |> map(
+        fn: (r) => ({r with
+            level_value: if r._level == levelCrit then
+                4
+            else if r._level == levelWarn then
+                3
+            else if r._level == levelInfo then
+                2
+            else if r._level == levelOK then
+                1
+            else
+                0,
+            foo: bar,
+        //Check indent on this test
+        }),
+    //Seems correct running this test
```

Comment:
Can we delete these two comments? This is another instance of something that's fine during development, but that we don't want to commit.

Reply:
Yes, sure but in that case should we also get rid of comments like this (starting line 723):
![Screen Shot 2021-05-18 at 3 01 05 PM](https://user-images.githubusercontent.com/53279816/118728995-f3c57880-b7e9-11eb-8a3f-938f7b07c0fe.png)

Reply:
I think those comments are fine because in that case they're flux comments and we're testing to see if comments are preserved when formatting. The comments are a bit tongue-in-cheek, but we can let that slide for now :stuck_out_tongue_closed_eyes:

Reply:
Oh, apologies. I think I misread your code. I see that your comments are comments in the flux code being tested. If the intention was to test the formatter behavior around comments, then they can stay. Otherwise, I think it's best to remove them.